src/rulesets/reader.ts

```diff
@@ -65,15 +65,11 @@ const createRulesetProcessor = (
         dereferenceInline: false,
         uriCache,
         async parseResolveResult(opts) {
-          try {
-            opts.result = parse(opts.result);
-          } catch {
-            // happens
-          }
-          return opts;
+          return { result: parse(opts.result) };
```

Comment:
If I am not mistaken, we should return original options.
I suggest not touch it unless it's needed?